### ISSUE TITLE
move to subword is somewhat naive

### ISSUE THREAD

**Issue by deathaxe** [robertcollier4/KeyboardNavigation#2]
While the possibility to move to the WORD boundaries which only respect whitespace seems useful in several situations, the `move_to_beg_of_subword_boundary` seems a bit naive.

1. If the caret is within a sequence of whitespace the command behaves just like an ordinary `move` command, which moves the caret forward or backward by one position. 

2. The command does not detect the subwords in _camelCase_ variables, which the native ST command does.

![Animation](https://user-images.githubusercontent.com/16542113/56884938-8233f980-6a6b-11e9-9a14-fd65632be364.gif)

**Comment by robertcollier4** [comment]
Good point, I do not believe in CamelCase which is why I did not include it. The subword separator symbols are changable in KeyboardNavigation.py but perhaps a option to change them more easily via JSON entry in .sublime-settings file could be included.
